src/sentry/incidents/subscription_processor.py

```diff
@@ -312,6 +312,14 @@ def get_comparison_aggregation_value(
 
         if not comparison_aggregate:
             metrics.incr("incidents.alert_rules.skipping_update_comparison_value_invalid")
+            logger.info(
+                "No comparison aggregate",
+                extra={
```

Comment:
We can check this to be sure this is where the None is coming from but I can't see any other possible place. As noted in Slack we need to pull this out of the subscription processor to not use `self.alert_rule` in a follow up.